packages/playground/website/src/main.tsx

```diff
@@ -269,6 +269,17 @@ function Main() {
 									>
 										Documentation
 									</MenuItem>
+									<MenuItem
+										icon={external}
+										iconPosition="left"
+										aria-label="Go to Playground documentation"
```

Comment:
This should be changed to something like _Go to the Playground git repository_

Reply:
Can I change it now? or should I again fork and make the change?